package.json

```diff
@@ -6,28 +6,33 @@
   "scripts": {
     "test": "npm run asbuild:untouched && npm run prettier:check && jest __tests__",
     "test:suite": "npm run asbuild:untouched && jest __spec_tests__ --reporter=jest-summary-reporter",
-    "jest": "jest __tests__",
+    "jest": "jest __tests__ --testPathIgnorePatterns assembly/*",
```

Comment:
should jest still be used now that you're moved to as-pect?

Reply:
I think it's okay to have both tests.

Reply:
But what tests will jest be running? doesn't this move all of the unit tests over to as-pect?

Reply:
Jest still tests the JS side.  So know you get internal unit tests and integration unit tests.